src/drivers/imx/ipc.c

```diff
@@ -190,6 +190,8 @@ int platform_ipc_init(struct ipc *ipc)
 	}
 #endif
 
+	interrupt_clear(PLATFORM_IPC_INTERRUPT);
```

Comment:
Will the interrupt retrigger if there is a pending message at this point? Or do we have a guarantee that no such pending message is possible?

Reply:
Shouldn't retrigger as MU registers are with reset values. We only need to clear internal DSP core state.